input/delimiter/fallback_delimiter.go

```diff
@@ -19,23 +19,23 @@ func init() {
 	runRegex = regexp.MustCompile(`\n` + SYSLOG_DELIMITER)
 }
 
-// A Delimiter detects when Syslog lines start.
-type Delimiter struct {
+// A FallbackDelimiter detects when Syslog lines start.
+type FallbackDelimiter struct {
```

Comment:
Calling this a "fallback delimiter" isn't right. It's just a particular type of delimiter. The policy that the code will fall back to this one is made at a higher-level in the code.

Let's call it the "syslog delimiter" or something similar. Make sense?